lib/view_model/send/send_view_model.dart

```diff
@@ -306,6 +318,29 @@ abstract class SendViewModelBase extends WalletChangeListenerViewModel with Stor
     }
   }
 
+  @action
+  Future<void> replaceByFee(String txId, String newFee) async {
+    state = IsExecutingState();
+
+    final isSufficient = await bitcoin!.isChangeSufficientForFee(wallet, txId, newFee);
+
+    if (!isSufficient) {
+      state = AwaitingConfirmationState(
+          title: "Confirm Fee Deduction",
+          message: "Do you agree to deduct the fee from the output?",
+          onConfirm: () async {
+            pendingTransaction = await bitcoin!.replaceByFee(wallet, txId, newFee);
+            state = ExecutedSuccessfullyState();
+          },
+          onCancel: () {
+            state = FailureState('Insufficient change for fee');
```

Comment:
translation
use this command to sort the added keys
`flutter packages pub run tool/append_translation.dart "hello_world" "Hello World!" `

Reply:
done